dashboard/app/templates.html

```diff
@@ -334,7 +334,7 @@ <h1><a href="/{{$.Namespace}}">syzbot</a></h1>
 			<td class="time">{{formatTime $b.Time}}</td>
 			<td class="kernel" title="{{$b.KernelAlias}}">{{$b.KernelAlias}}</td>
 			<td class="tag" title="{{$b.KernelCommit}}
-	{{formatTime $b.KernelCommitDate}}">{{link $b.KernelCommitLink (formatShortHash $b.KernelCommit)}}</td>
+	{{formatTime $b.KernelCommitDate}}">{{link $b.KernelCommitLink (formatTagHash $b.KernelCommit)}}</td>
```

Comment:
Is formatShortHash still used anywhere? If not, remove it. Somebody may use it in future just because it exists.

Reply:
It is still used for syzkaller commit hashes